### DISCUSSION TITLE
[Bug report] Backup: Wrong creation date shown

### DISCUSSION THREAD

**Discussion by bttrx** [GetPublii/Publii#1916] [Bug Report] [answered]
### Operating system

Linux Mint 22 MATE

### Publii version

0.46.3 (build 17008)

### Issue type

Application

### Bug description

'Created at' doesn't match the real creation date. Except the latest backup all previous backups show the same creation date. (Also unlocalized.)

![grafik](https://github.com/user-attachments/assets/a2850896-20dc-45ee-a574-55f43fa09b34)

Could switching from `Publii-0.46.2.AppImage` to `Publii-0.46.3.AppImage` be the reason?

As shown in Double Commander:

![grafik](https://github.com/user-attachments/assets/7b632bd0-cb2c-40e7-820f-7f3063070dc6)

**Comment by dziudek** [comment, marked as answer]
@bttrx thanks for pointing this issue - the wrong file param has been used to display dates - in v.0.46.4 it will be fixed
